notebooks/0110_cmip-conc-inversions.py

```diff
@@ -18,6 +18,16 @@
 # Invert concentrations from CMIP7.
 # We do this for a very limited number of species
 # where we have no other obvious option.
+#
+# Note on N2O: Global N2O budget is reported in units
+# of TgN / yr, but the unit is really Tg N2 / yr. The
+# clearest evidence of this is the Supplement to
+# Global N2O budget Table 3, where
+# 44 / 28 = 1.57
+# (https://essd.copernicus.org/articles/16/2543/2024/essd-16-2543-2024-supplement.pdf)
+#
+# We calculate 1750 total N2O emissions to be in the region of 19 Tg N2O / yr from the inversion,
+# which is close to the middle of the natural range from GNB (natural sources 11.8 TgN2 / yr = 18.5 Tg N2O / yr).
```

Comment:
```suggestion

```
I'd suggest moving this down to a cell that just considers N2O. I've been confused what it means every time I read it.